.gitignore

```diff
@@ -1,3 +1,4 @@
 target
 *.bk
 Cargo.lock
+.idea/
```

Comment:
This kind of pattern belongs in a global gitignore:

> Patterns which a user wants Git to ignore in all situations (e.g., backup or temporary files generated by the user’s editor of choice) generally go into a file specified by core.excludesFile in the user’s ~/.gitconfig. Its default value is $XDG_CONFIG_HOME/git/ignore. If $XDG_CONFIG_HOME is either not set or empty, $HOME/.config/git/ignore is used instead.

<https://git-scm.com/docs/gitignore>